ino/environment.py

```diff
@@ -79,7 +79,12 @@ class Environment(dict):
     ]
 
     if platform.system() == 'Darwin':
-        arduino_dist_dir_guesses.insert(0, '/Applications/Arduino.app/Contents/Resources/Java')
+        if os.path.exists('/Applications/Arduino.app/Contents/Java'):
+            arduino_dist_dir_guesees.insert(0, '/Applications/Arduino.app/Contents/Java')
```

Comment:
There is a typo in `quesees` instead of `guesses`

Reply:
On line 83 just before the insert function call @ipinak